test/src/main/java/org/openjdk/skara/test/TestHostedRepository.java

```diff
@@ -56,7 +56,7 @@ public TestHostedRepository(TestHost host, String projectName, Repository localR
         this.host = host;
         this.projectName = projectName;
         this.localRepository = localRepository;
-        pullRequestPattern = Pattern.compile(url().toString() + "/pr/" + "(\\d+)");
+        pullRequestPattern = Pattern.compile(authenticatedUrl().toString() + "/pr/" + "(\\d+)");
```

Comment:
Doesn't matter much as they resolve to the same value, but logically I think `.webUrl()` fits better here.